frontend/src/app/shared/components/user-notification/user-notification.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { NotificationsHubService } from '@core/hubs/notifications-hub.service';
+import { HttpService } from '@core/services/http.service';
+import { INotification } from '@shared/models/INotification';
+import { Subscription } from 'rxjs';
+
+@Component({
+    selector: 'app-user-notification',
+    templateUrl: './user-notification.component.html',
+    styleUrls: ['./user-notification.component.sass'],
+})
+export class UserNotificationComponent implements OnInit, OnDestroy {
+    readonly notificationsHub: NotificationsHubService;
+
+    readonly httpService: HttpService;
+
+    notifications: INotification[] = [];
+
+    notifySubscription: Subscription;
+
+    constructor(notificationsHub: NotificationsHubService, httpService: HttpService) {
+        this.notificationsHub = notificationsHub;
+        this.httpService = httpService;
+    }
+
+    async ngOnInit() {
+        this.notifySubscription = await this.httpService.get<INotification>('/notifications').subscribe((data) => {
+            this.notifications = data;
+        });
+
+        await this.notificationsHub.start();
+
+        const storedUserData = localStorage.getItem('user');
+        let userEmail: string = 'Test@test.ua';
+
+        if (storedUserData) {
+            userEmail = JSON.parse(storedUserData).email.toLowerCase();
+        }
+
+        this.notificationsHub.listenMessages((msg) => {
+            const broadcastMessage: INotification = JSON.parse(msg.toLowerCase());
+
+            if (broadcastMessage.email === userEmail.toLowerCase()) {
+                this.notifications.push(broadcastMessage);
+            }
```

Comment:
It's better to implement in the way, that you will pass parameters on connection start and then send from backend only to those users, who needs this